doc/sphinx/user-extensions/syntax-extensions.rst

```diff
@@ -1373,6 +1373,9 @@ interpreted in the scope stack extended with the scope bound to :n:`@scope_key`.
 
    Removes the delimiting keys associated with a scope.
 
+The arguments of an :ref:`abbreviation <Abbreviations>` can be interpreted
+in a given scope by using the modifier :n:`{+, @ident } in scope @scope_name`.
```

Comment:
Please make it clear whether this is about a temporary scope (applies only to the top-most construction) or part of the scope stack (persists into subterms)

Reply:
I'm not sure @herbelin , the annotation has the same "persistence" than the one that is automatically inferred, but I don't know which one it is.

Reply:
done and tested